client/login/wp-login/index.jsx

```diff
@@ -104,12 +104,16 @@ class Login extends React.Component {
 				</a>;
 		}
 
-		const goBackLink = ! magicLoginView && <a
-			href="#"
-			key="back-link"
-			onClick={ this.goBack }>
-				<Gridicon icon="arrow-left" size={ 18 } /> { this.props.translate( 'Return' ) }
-			</a>;
+		let goBackLink;
+		if ( typeof window === 'undefined' || window.history.length > 1 ) {
```

Comment:
Not sure I understand that, why `||` and not `&&` ?
`this.goBack` is essentially `window.history.back();` so if we don't have a `window` we shouldn't display a link as well, am I missing something?